config/environments/production.rb

```diff
@@ -1,5 +1,3 @@
-require Rails.root.join("config", "secret") if Rails.root.join("config", "secret.rb").file?
```

Comment:
Yeah it's were we have all the real secrets. This is leftover from long ago when `secrets.yml` didn't exist. We need this for now but it's something we can refactor in the future.